lib/drivers/direct/index.js

```diff
@@ -23,6 +23,7 @@ function Driver(options) {
   this._baseDir = mandatory(options.baseDir);
   this._console = mandatory(options.console);
   this._server = mandatory(options.server);
+  this._express = options.express;
```

Comment:
In answer to your question: the express server has been passed here to the direct driver, but it is unused other than being assigned to a property.

I assume it will be passed to something else, the runner eventually, so that it can be used to establish a control channel?

Feel free to ignore me, I'm just musing. I'll see what you intend when you are ready for review.